app.js

```diff
@@ -83,7 +78,6 @@ function sumArray(testArray){ //eslint-disable-line
   var sumAnswer = secondSum[0];
   var sumMessage = testArray[0] + ',' + testArray[1] + ',' + testArray[2] + ' was passed in as an array of numbers, and ' + sumAnswer + ' is their sum.';
   var totalSumArray = [sumAnswer, sumMessage];
```

Comment:
this variable is only used to return the array, return can be used in place of a variable declaration.